hathor/cli/run_node.py

```diff
@@ -100,6 +100,7 @@ def create_parser(cls) -> ArgumentParser:
         parser.add_argument('--x-localhost-only', action='store_true', help='Only connect to peers on localhost')
         parser.add_argument('--x-rocksdb-indexes', action='store_true', help=SUPPRESS)
         parser.add_argument('--x-enable-event-queue', action='store_true', help='Enable event queue mechanism')
+        parser.add_argument('--x-reset-event-queue', action='store_true', help='Reset the event queue')
```

Comment:
I'm not sure I like doing it this way. It seems too easy to forget this flag in the `Dockerfile` and wipe events out by mistake. Maybe a cli command (e.g., `./hathor-cli reset-event-queue`) is safer. What do you think?

Reply:
Oh, i hadn't thought about that. Great suggestion. Changed in https://github.com/HathorNetwork/hathor-core/pull/608/commits/79f2773a773adcbef4f5d853344e51bb66cfb4d2

Reply:
Great! Thanks!